### PR TITLE
Layouts for Blogs displayed on Content or Full Page Application pages not found with BasePortletLayoutFinder

### PR DESCRIPTION
**Pull request by SamZiemer**
https://issues.liferay.com/browse/LPS-182722

When a blog aggregator links to content on a Content or full page application type layout, those layouts are not found because the BasePortletLayoutFinder only retrieves Widget pages.

**Proposed Solution**
Add the other layout types that widgets can be placed on and scoped to.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java [hunk 1]
```diff
@@ -239,9 +239,15 @@ private ObjectValuePair<Long, String> _getPlidPortletIdObjectValuePair(
...
+		String[] portletTypes = {
+			LayoutConstants.TYPE_CONTENT,
+			LayoutConstants.TYPE_FULL_PAGE_APPLICATION,
+			LayoutConstants.TYPE_PORTLET
+		};
+
 		for (boolean privateLayout : Arrays.asList(false, true)) {
 			List<Layout> layouts = LayoutLocalServiceUtil.getLayouts(
-				groupId, privateLayout, LayoutConstants.TYPE_PORTLET);
+				groupId, privateLayout, portletTypes);
```
Comment (AliciaGarciaGarcia) [inline portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java:250]: Hi @SamZiemer 

this kind of call is normally inlined as :
```

new String[] {
			LayoutConstants.TYPE_CONTENT,
			LayoutConstants.TYPE_FULL_PAGE_APPLICATION,
			LayoutConstants.TYPE_PORTLET
	}
```

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java (hunk 1 above)
Comment (SamZiemer) [reply portal-kernel/src/com/liferay/portal/kernel/portlet/BasePortletLayoutFinder.java]: Hi @AliciaGarciaGarcia , I've pushed another commit to inline this, thank you.

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The above failures are not related to this PR.

**Comment by Hanlf** [comment]
ci:forward:force

**Pull request closed without merging**